docs/Userguide_multinode.rst

```diff
@@ -0,0 +1,166 @@
+Multiple Nodes
+==============
+
+Data Parallel
+-------------
+
+.. image:: _static/dataparallel.png
+
+Request 3 nodes with at least 4 GPUs each.
+
+.. code-block:: bash
+   :linenos:
+
+   #!/bin/bash
+
+   # Number of Nodes
+   #SBATCH --nodes=3
+
+   # Number of tasks. 3 (1 per node)
+   #SBATCH --ntasks=3
+
+   # Number of GPU per node
+   #SBATCH --gres=gpu:4
+   #SBATCH --gpus-per-node=4
```

Comment:
I think the `--gres=gpu:4` is not needed since `--gpus-per-node` is used but that would not break anything

```suggestion
   # Number of GPU per node
   #SBATCH --gpus-per-node=4
```

Reply:
@obilaniu is `--gres=gpu` needed when `--gpus-per-node` is used? In the doc it seams that it is not https://slurm.schedmd.com/gres.html#Running_Jobs